crates/ruff/src/message/snapshots/ruff__message__json__tests__output.snap

```diff
@@ -25,7 +25,7 @@ expression: content
       "message": "Remove assignment to unused variable `x`",
       "edits": [
         {
-          "content": "",
+          "content": null,
```

Comment:
Using `Option` results in `content` being null for `Deletions`. Is this a problem for the LSP (it's easy to fix, but I prefer `null`  over `""`

Reply:
Yeah I think it will be a problem for the LSP. This is ultimately passed into the LSP's `TextEdit` object which has the following schema:

```py
@attrs.define
class TextEdit:
    """A text edit applicable to a text document."""

    range: "Range" = attrs.field()
    """The range of the text document to be manipulated.

    To insert text into a document create a range where start === end.
    """

    new_text: str = attrs.field(validator=attrs.validators.instance_of(str))
    """The string to be inserted.

    For delete operations use an empty string.
    """
```

Reply:
So we can either revert this, or change the LSP to respect it (`new_text=edit["content"] or ""`), or change the JSON serialization to do this mapping for us.

Reply:
Let me revert this. We can change it in another pr if we want